README.md

```diff
@@ -709,6 +710,49 @@ Google Stackdriver Monitoring can be installed separately by requiring the `goog
 $ composer require google/cloud-monitoring
 ```
 
+## Google Cloud Dataproc (Beta)
+
+- [API Documentation](http://googlecloudplatform.github.io/google-cloud-php/#/docs/latest/dataproc/readme)
+- [Official Documentation](https://cloud.google.com/dataproc/docs)
+
+#### Preview
+
+```php
+require 'vendor/autoload.php';
```

Comment:
The require here should be below `<?php`

Reply:
Fixed